Makefile

```diff
@@ -62,7 +62,9 @@ _touch_pot:
 	fi
 	touch $@
 
-# Usage: make bump updateType=patch
+# Usage:
+# make bump updateType=patch
+# make bump updateType=patch bumpRc=true
```

Comment:
Can we have the examples of the expected changes?

For instance:
```
`3.0.1` -> make bump updateType=major --bumpRc -> `4.0.0`
`3.0.1` -> make bump updateType=minor --bumpRc -> `3.1.0-rc.1`
`3.2.1` -> make bump updateType=patch -> `3.2.2`
`3.2.1-rc.1` -> make bump updateType=release -> `3.2.1`
```

Reply:
Great suggestion! Implemented both on comments and on the error message on https://github.com/HathorNetwork/hathor-wallet-mobile/pull/652/commits/b16d0161ba30e2446bb010ae0a0d6b23e38e691d